Client/src/style/_buttons.scss

```diff
@@ -0,0 +1,44 @@
+@import 'colors.scss';
+
+button{
+    font-family: 'Roboto';
+    letter-spacing: normal;
+    padding-left: 16px;
+    padding-right: 16px;
+    padding-top: 8px;
+    padding-bottom: 8px;
+    border-radius: 5px;
```

Comment:
`padding-left: 16px;
    padding-right: 16px;
    padding-top: 8px;
    padding-bottom: 8px;`
Można to przerobić na 

`padding: 8px 16px`

Reply:
Poprawione